packages/mail-swiftmailer/src/Jellyfish/MailSwiftmailer/MailSwiftmailerFactory.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Jellyfish\MailSwiftmailer;
+
+use Jellyfish\Config\ConfigFacadeInterface;
+use Jellyfish\Mail\MailConstants;
+use Swift_Mailer as SwiftMailer;
+use Swift_Message as SwiftMessage;
+use Swift_SmtpTransport as SwiftSmtpTransport;
+use Swift_Transport as SwiftTransport;
+
+class MailSwiftmailerFactory
+{
+    /**
+     * @var \Jellyfish\Config\ConfigFacadeInterface
+     */
+    protected ConfigFacadeInterface $configFacade;
+
+    /**
+     * @var \Swift_Mailer|null
+     */
+    protected ?SwiftMailer $swiftMailer = null;
+
+    /**
+     * @var \Swift_Transport|null
+     */
+    protected ?SwiftTransport $swiftTransport = null;
+
+    /**
+     * @var \Jellyfish\MailSwiftmailer\SwiftAttachmentFactory|null
+     */
+    protected ?SwiftAttachmentFactory $swiftAttachmentFactory = null;
+
+    /**
+     * @param \Jellyfish\Config\ConfigFacadeInterface $configFacade
+     */
+    public function __construct(ConfigFacadeInterface $configFacade)
```

Comment:
I think its better to use one mail config instead of global config and make only needed config params available